src/ui/core.c

```diff
@@ -89,10 +89,26 @@ static Display* display;
 
 static void _ui_draw_term_title(void);
 
+static void
+_ui_close(void)
+{
+    g_timer_destroy(ui_idle_time);
+    endwin();
```

Comment:
I see this was already earlier the case. But now I'm thinking: shouldn't endwin() be the last ncurses function to call?

Reply:
`man endwin` says `Call delscreen after endwin when a particular SCREEN structure is no longer needed.`